kong/clustering/version_negotiation/services_known.lua

```diff
@@ -0,0 +1,6 @@
+return {
+  cluster_protocol = {
+    { version = "json", message = "current" },
+    { version = "wrpc", message = "beta" },
```

Comment:
The version was assumed to be "v1", "v2" but wasn't clarified in the spec.
Can we keep "v0" as what exists today and then "v1" as the RPC based protocol?

Reply:
done